src/jdk.jdwp.agent/share/native/libjdwp/eventHelper.c

```diff
@@ -605,6 +623,35 @@ handleReportVMInitCommand(JNIEnv* env, ReportVMInitCommand *command)
     /* Why aren't we tossing this: tossGlobalRef(env, &(command->thread)); */
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void
+handleReportVMRestoreCommand(JNIEnv* env, ReportVMRestoreCommand *command)
+{
+    PacketOutputStream out;
```

Comment:
```suggestion
    PacketOutputStream out = {0};
```

Reply:
Same as above, reverted change in https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/1396459ab0d1f728e71ed55dd10af33c53c2c8b7 since it will get memset.